### ISSUE TITLE
TMD3.0 prediction problem

### ISSUE THREAD

**Issue by liuyalong008** [chadagreene/Tide-Model-Driver#23]
The interpolation in TMD3.0 may have issues. For example, at a point (122.5, 38), the constituents obtained by interpolation one by one are:
6 - 2i
0.006 - 0.002i
-0.1950001 - 0.1029999i
0.00799991 + 0.04799991i
0.4889991 + 0.3999993i
-0.014 + 0.01600009i
-0.004 - 0.003i
-0.002 + 0i
-0.001 + 0.004i
-0.006 + 0i
0.1179998 + 0.02400001i
-0.141 + 0.01600018i
-0.06200004 - 0.03099996i
-0.021 + 0.002000086i
-0.002 + 0i
0.02599994 + 0.1819996i

And setting breakpoints in TMD yields:
0.012510936552071 + 0.000893638325147925i
-0.19615361236997 - 0.083778592982618i
0.000670228743860944 + 0.0457989641638312i
0.499097004595116 + 0.407275666686167i
-0.0127343461333579 + 0.012064117389497i
-0.00558523953217453 - 0.00268091497544378i
0.00178727665029585 - 0.000670228743860944i
-0.00692569701989642 + 0.00469160120702661i
-0.00826615450761831 - 0.00647887785732246i
0.11393888645636 + 0.0308305222176034i
-0.122428450545266 + 0.0279261976608727i
-0.0513842036960057 - 0.0223409581286981i
-0.0172025377590976 + 0.000893638325147925i
-0.00111704790643491 - 0.000446819162573963i
0.0216707293848372 + 0.192802468650665i
This can result in a tidal difference of several cm.

![TMD3_issue](https://github.com/user-attachments/assets/7265007d-0eab-4c14-b7a2-90d70cfb5092)

**Comment by chadagreene** [comment]
Hmm it's hard to follow exactly what you mean. Can you provide a minimal working example to replicate the figure?

**Comment by liuyalong008** [comment]
I apologize, upon further testing, it was found to be the difference between TPXO9 and TPXO10.
